src/TMap.h

```diff
@@ -111,34 +114,42 @@ class TMap : public QObject
     void initGraph();
     void connectExitStub(int roomId, int dirType);
     void postMessage(const QString text);
-    void set3DViewCenter(const int, const int, const int, const int);
+
     // Used by the 2D mapper to send view center coordinates to 3D one
+    void set3DViewCenter(const int, const int, const int, const int);
 
     void appendRoomErrorMsg(const int, const QString, const bool isToSetFileViewingRecommended = false);
     void appendAreaErrorMsg(const int, const QString, const bool isToSetFileViewingRecommended = false);
     void appendErrorMsg(const QString, const bool isToSetFileViewingRecommended = false);
     void appendErrorMsgWithNoLf(const QString, const bool isToSetFileViewingRecommended = false);
-    void pushErrorMessagesToFile(const QString, const bool isACleanup = false);
+
     // If the argument is true does not write out any thing if there is no data
     // to dump, intended to be used before an operation like a map load so that
     // any messages previously recorded are not associated with a "fresh" batch
     // from the operation.
+    void pushErrorMessagesToFile(const QString, const bool isACleanup = false);
 
     // Moved and revised from dlgMapper:
     void downloadMap(const QString* remoteUrl = Q_NULLPTR, const QString* localFileName = Q_NULLPTR);
+
     // Also uses readXmlMapFile(...) but for local files:
     bool importMap(QFile&, QString* errMsg = Q_NULLPTR);
+
     // Used at end of downloadMap(...) OR as part of importMap(...) but not by
     // both at the same time thanks to mXmlImportMutex
     bool readXmlMapFile(QFile&, QString* errMsg = Q_NULLPTR);
-    // Use progresss dialog for post-download operations:
+
+    // Use progress dialog for post-download operations.
     void reportStringToProgressDialog(const QString);
+
+    // Use progress dialog for post-download operations.
```

Comment:
I'd like to eventually turn on clang's `-Wdocumentation` warnings so we can have proper docs for every function. That's probably a long ways away, but part of it is that each function has it's own docs. I'd rather not rely on one comment referring to two functions, since that's unexpected/surprising, in general.